pkg/checkpoint/manifest.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 
 	"github.com/golang/glog"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
can we use `corev1` instead of `v1`. `corev1` is present in other parts of the codebase already and helps with differentiating with `meta/v1` etc

Reply:
Done.